### ISSUE TITLE
[Accessibility]: Mobile - Chats are not being read properly by screen-readers

### ISSUE THREAD

**Issue by sunnil24** [highcharts/highcharts#16742]
#### Expected behavior
Charts should be A11Y compatible on mobile devices too.

#### Actual behaviour
Chats are not being read properly by mobile screen-readers like VO[voice-over] and talk-back. Following are the issues while navigation through VO:
1. Content is not reading in proper order.
2. Focus on a graph is incorrect
3. Blank spaces are getting called out
4. Long press and double tap on text field is also getting called out

#### Live demo with steps to reproduce
1- Open https://jsfiddle.net/highcharts/LLExL/ on iPhone in Safari
2- Enable voice over and start reading the content by doing right swipe or left swift in order to read different elements

#### Product version
All

#### Affected browser(s)
All

**Comment by raf18seb** [comment]
Hi @sunnil24, thank you for creating the ticket, we will take a look at it soon.

**Comment by oysteinmoseng** [comment]
Hi @sunnil24, you need to include the `accessibility.js` module for accessibility features in Highcharts. Here is a link to the relevant documentation and how to configure it for your use case: https://www.highcharts.com/docs/accessibility/accessibility-module

Closing this for now, feel free to comment if there was a misunderstanding, and we can reopen.
